xml/snmp_sysdescr.xml

```diff
@@ -5672,11 +5672,11 @@ Copyright (c) 1995-2005 by Cisco Systems
    =======================================================================-->
 
   <fingerprint pattern="^PARADYNE T1 DSU/CSU; model: ([^;]+); S/W Release: ([^;]+);">
-    <description>Paradyne CSU/DSU</description>
+    <description>Paradyne DSU/CSU</description>
```

Comment:
Here and below. Simple typo as can been seen in the example. Changed to align with correct usage elsewhere.